pkg/apis/zookeeper/v1beta1/zookeepercluster_types_test.go

```diff
@@ -241,6 +242,42 @@ var _ = Describe("ZookeeperCluster Types", func() {
 			Ω(p.Leader).To(BeEquivalentTo(3888))
 		})
 
+		It("should have a metrics port", func() {
+			Ω(p.Metrics).To(BeEquivalentTo(7000))
+		})
+	})
+	Context("#ZookeeperPorts with values set", func() {
+		var p v1beta1.Ports
+		BeforeEach(func() {
+			ports := []v1.ContainerPort{
+				{
+					Name:          "testclient",
+					ContainerPort: 2181,
+				},
+			}
+			z = v1beta1.ZookeeperCluster{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "example",
+				},
+				Spec: v1beta1.ZookeeperClusterSpec{
+					Ports: ports,
+				},
+			}
+			z.WithDefaults()
+			p = z.ZookeeperPorts()
+		})
+		It("should have a client port", func() {
```

Comment:
we can remove this assertion as we are only setting it and it's no set by the "withDefaults()" function.

Reply:
we are creating a dummy value and setting defaults so that all the conditions in else is getting executed.

Reply:
okay